swift/ios-shoppe-demo/Views/CartTableViewController.swift

```diff
@@ -32,7 +32,7 @@ class CartTableViewController: UITableViewController {
     }
 
     func setOrderItemsForCart() {
-        items = order.items.filter { $0.quantity > 0 }
+        items = order.getFilteredItems()
```

Comment:
while looking at potentially changing `order.items` into a map, I ended up removing any direct interaction with `order.items` from controllers and made some functions to encapsulate `Order`. But open to discussion on swift norm here